src/graphql/root/mutation/account-data-update.ts

```diff
@@ -0,0 +1,60 @@
+import { getAccountsConfig } from "@config"
+
+import { Accounts } from "@app"
+
+import { GT } from "@graphql/index"
+import { mapError } from "@graphql/error-map"
+import { parseCustomFieldsSchema } from "@graphql/helpers"
+import AccountDataUpdatePayload from "@graphql/types/payload/account-data"
+
+const customFieldsSchema = getAccountsConfig().customFields
+
+const AccountDataUpdateInput = GT.Input({
+  name: "AccountDataUpdateInput",
+  fields: () => {
+    const customFields = parseCustomFieldsSchema(customFieldsSchema)
+
+    return {
+      transactionsCallback: { type: GT.String },
```

Comment:
If ```transactioonsCallback``` is something we want present on all accounts, I think it should be separated from the customFields in how it is updated because it is not conceptually the same as customer specifc metadata and will impact behavior in the backend where as other metadata does not effect any behavior of the backend. Additionally, custom fields can override ```transactionsCallback``` since it is destructured after ```transactionsCallback``` is defined.

Reply:
done https://github.com/GaloyMoney/galoy/pull/1465/commits/c882fa2a11dbbb176ea1eff41e61cd8e6b702fbf